SamMou.Api.Graph/GraphQLSchema.cs

```diff
@@ -9,9 +9,10 @@ namespace SamMou.Api.GraphQL
 {
     public class GraphQLSchema : Schema
     {
-        public GraphQLSchema(IServiceProvider services, GraphQLQuery query) : base(services)
+        public GraphQLSchema(QueryObject query, MutationObject mutation, IServiceProvider sp) : base(sp)
```

Comment:
where is QueryObject and MutationObject coming from? is it because you've added scope into the app [here](https://github.com/moumot/SamMou/blob/3c16ba49c9a7eca4babad4950df08fa8cbae9ba0/SamMou.Api/Startup.cs#L41-L42)

Reply:
yup this is dependency injection